src/update_engine/update_attempter_unittest.cc

```diff
@@ -338,7 +342,7 @@ void UpdateAttempterTest::UpdateTestVerify() {
   EXPECT_EQ(attempter_.response_handler_action_.get(),
             attempter_.actions_[1].get());
   DownloadAction* download_action =
-      dynamic_cast<DownloadAction*>(attempter_.actions_[4].get());
+      dynamic_cast<DownloadAction*>(attempter_.actions_[5].get());
```

Comment:
Eeek. I mean, I get it, but would it be possible to, as a comment, point me to the place (above) where we're getting this index from?